Example/Controllers/ViewController.swift

```diff
@@ -103,10 +103,10 @@ class ViewController: UIViewController {
     }
 
     func compareMiniAppMetaData(miniAppInfo: MiniAppInfo, completionHandler: @escaping (Result<Bool, Error>) -> Void) {
-        guard let downloadedManifest = MiniApp.shared().getDownloadedManifest(miniAppId: miniAppInfo.id) else {
+        guard let downloadedManifest = MiniApp.shared(with: Config.getCurrent()).getDownloadedManifest(miniAppId: miniAppInfo.id) else {
             return completionHandler(.success(true))
         }
-        MiniApp.shared().getMiniAppManifest(miniAppId: miniAppInfo.id, miniAppVersion: miniAppInfo.version.versionId) { (result) in
+        MiniApp.shared(with: Config.getCurrent()).getMiniAppManifest(miniAppId: miniAppInfo.id, miniAppVersion: miniAppInfo.version.versionId) { (result) in
```

Comment:
do you think we should rename `getCurrent` to just `current`?

Reply:
I am thinking of `getCurrentConfig`  or may be `currentConfig` 🤔 . But I don't want this refactor to be pushed before the release. But I will add this in To-do